<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotePad</class>
 <widget class="QMainWindow" name="NotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1041</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="color">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>More</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bold_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/bold_icon.png</normaloff>:/imgs/imgs/bold_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bold_button_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/bold_icon.png</normaloff>:/imgs/imgs/bold_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="underline_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/underline_icon.png</normaloff>:/imgs/imgs/underline_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="underline_button_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/underline_icon.png</normaloff>:/imgs/imgs/underline_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="center_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/align_center_icon.png</normaloff>:/imgs/imgs/align_center_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="italic_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/italic_icon.png</normaloff>:/imgs/imgs/italic_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="center_button_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/align_center_icon.png</normaloff>:/imgs/imgs/align_center_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="italic_button_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/italic_icon.png</normaloff>:/imgs/imgs/italic_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="linethrough">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/linethrough_icon.png</normaloff>:/imgs/imgs/linethrough_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="linethrough_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/linethrough_icon.png</normaloff>:/imgs/imgs/linethrough_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="paste_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/paste_icon.png</normaloff>:/imgs/imgs/paste_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/copy_icon.png</normaloff>:/imgs/imgs/copy_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cut_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/cut_icon.png</normaloff>:/imgs/imgs/cut_icon.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="case_combo">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Lowercase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Uppercase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sentence</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="align_right_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/align_right_icon.png</normaloff>:/imgs/imgs/align_right_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="align_right_2_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/align_right_icon.png</normaloff>:/imgs/imgs/align_right_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="align_left_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/align_left_icon.png</normaloff>:/imgs/imgs/align_left_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="align_left_button_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/imgs/imgs/align_left_icon.png</normaloff>:/imgs/imgs/align_left_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <zorder>align_left_button_2</zorder>
   <zorder>align_right_2_button</zorder>
   <zorder>linethrough_2</zorder>
   <zorder>italic_button_2</zorder>
   <zorder>center_button_2</zorder>
   <zorder>underline_button_2</zorder>
   <zorder>bold_button_2</zorder>
   <zorder>italic_button</zorder>
   <zorder>center_button</zorder>
   <zorder>underline_button</zorder>
   <zorder>bold_button</zorder>
   <zorder>textEdit</zorder>
   <zorder>fontComboBox</zorder>
   <zorder>comboBox</zorder>
   <zorder>color</zorder>
   <zorder>linethrough</zorder>
   <zorder>paste_button</zorder>
   <zorder>copy_button</zorder>
   <zorder>cut_button</zorder>
   <zorder>lineEdit</zorder>
   <zorder>case_combo</zorder>
   <zorder>align_right_button</zorder>
   <zorder>align_left_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">QMenu
{
	width: 200px;
	background-color: white;
}</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHome"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
